dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java

```diff
@@ -524,7 +524,6 @@ protected void copySettingsAndProfile(
             RuntimeUtil.loadProperties(prop2, profile);
         }
         prop2.putAll(prop);
-        prepareApplicationProperties(prop2);
```

Comment:
This call still needs to be there

Reply:
It's used just here: https://github.com/apache/camel/pull/11528/files

For adding properties, which will be already exported by default from application.properties.. I introduced the method myself and I don't think it's needed. The prop2.putAll will do all the needed.

Reply:
I'll check again. I'm missing something I guess.

Reply:
From the code then ExportQuarkus needs to have this method called as it does something special

Reply:
It should be fine now.